app/models/reservation.rb

```diff
@@ -121,7 +121,15 @@ def self.relay_in_progress
     where("actual_start_at IS NOT NULL AND actual_end_at IS NULL")
   end
 
-  scope :not_offline, -> { where(type: nil) }
+  def self.upcoming_offline(start_at_limit = 1.day.from_now)
```

Comment:
Where else will this be used? I almost think that the default should be now and the service object should be in charge of deciding how far out to look.